### PR TITLE
Correct check for open generic

### PR DESCRIPTION
**Pull request by StijnVA**
Using IsGenericTypeDefinition instead of IsGenericType to only register as open generic if it is an actual open generic and not as soon it is a type with a type parameter.

### CODE REVIEW

File: src/Autofac.Extras.AggregateService/AggregateServicesResources.resx [hunk 1]
```diff
@@ -120,4 +120,8 @@
   <data name="TypeMustBeInterface" xml:space="preserve">
     <value>Aggregate service type must be an interface.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="InterfaceMayNotContainGenericParameters" xml:space="preserve">
+    <value>Aggregate service type may not have an open generic type parameter.</value>
+  </data>
+
```
Comment (tillig) [inline src/Autofac.Extras.AggregateService/AggregateServicesResources.resx:126]:
```suggestion
```

File: test/Autofac.Extras.AggregateService.Test/AggregateServiceOpenGenericRegressionFixture.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Autofac Project. All rights reserved.
```
Comment (tillig) [inline test/Autofac.Extras.AggregateService.Test/AggregateServiceOpenGenericRegressionFixture.cs]: Can we put all the open generic tests in one fixture?

**Review by tillig — CHANGES_REQUESTED** [review]
A couple of minor things and I'll pull this for a 0.0.1 increment bug fix release. Thanks for the help!

File: test/Autofac.Extras.AggregateService.Test/AggregateServiceOpenGenericRegressionFixture.cs (hunk 2 above)
Comment (StijnVA) [reply test/Autofac.Extras.AggregateService.Test/AggregateServiceOpenGenericRegressionFixture.cs]: Done.

**Review by tillig — APPROVED** [review]
🎉

**Comment by tillig** [comment]
I can get that whiltespace thing when I update the semver.

**Pull request merged**